pkg/vault/contracts/test/BaseHooksMock.sol

```diff
@@ -5,10 +5,9 @@ pragma solidity ^0.8.24;
 import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
 import { IBasePool } from "@balancer-labs/v3-interfaces/contracts/vault/IBasePool.sol";
 import {
-    AddLiquidityKind,
-    HooksConfig,
     HookFlags,
     LiquidityManagement,
+    AddLiquidityKind,
     RemoveLiquidityKind,
     TokenConfig
 } from "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
```

Comment:
Since this is a mock, and not in the pool-hooks example pkg, maybe it should be a one-liner? Not sure.

Reply:
Yeah; these are judgement calls, but I think this one can be collapsed. (I mainly don't want to "hide" ERC20 or IRateProvider imports, and that doesn't apply here.)